cnd/Cargo.toml

```diff
@@ -35,6 +36,7 @@ libp2p-comit = { path = "../libp2p-comit" }
 libsqlite3-sys = { version = ">=0.8.0, <0.13.0", features = ["bundled"] }
 log = { version = "0.4", features = ["serde"] }
 lru = "0.4.3"
+multihash = "0.10" # TODO: Upgrade libp2p and use libp2p::multihash instead
```

Comment:
:mega: This TODO needs to be removed.

Reply:
Man I deleted that little sucker about 10 times during the `ln-single-patch` branch - how did it get back in :)